home work 25/server/src/database/index.js

```diff
@@ -0,0 +1,14 @@
+import { join, dirname } from 'node:path'
+import { fileURLToPath } from 'node:url'
+
+import { Low } from 'lowdb'
+import { JSONFile } from 'lowdb/node'
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const file = join(__dirname, 'db.json')
+// Configure lowdb to write to JSONFile
+const adapter = new JSONFile(file)
+const db = new Low(adapter)
+
+await db.read()
```

Comment:
don't use `await` on top level of code.
Here's the only example when you can use it on top level [top_level_await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await#top_level_await)